CMakeLists.txt

```diff
@@ -91,12 +91,14 @@ pkg_check_modules(Jansson REQUIRED jansson)
 find_package(libhandlegraph)
 if (${libhandlegraph_FOUND})
     message("Using installed libhandlegraph")
-else()
+elseif (NOT TARGET handlegraph_objs)
     message("Using bundled libhandlegraph")
     add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/deps/libhandlegraph")
-    add_library(libhandlegraph::handlegraph_shared ALIAS handlegraph_shared)
-    add_library(libhandlegraph::handlegraph_static ALIAS handlegraph_static)
+else ()
+    message("Using libhandlegraph built by someone else")
 endif()
+add_library(libhandlegraph::handlegraph_shared ALIAS handlegraph_shared)
+add_library(libhandlegraph::handlegraph_static ALIAS handlegraph_static)
```

Comment:
At some point we might want to revise the `libhandlegraph` CMake configuration to define scoped targets like this itself, instead of making aliases of them here.